packages/contracts/mud.config.ts

```diff
@@ -26,9 +30,18 @@ export default defineWorld({
                 dataStruct: true
             }
         },
+        MaterialMetadata: {
+            key: ["materialId"],
+            schema: {
+                materialId: "MaterialId",
+                tokenAddress: "address",
+                difficultyCoefficient: "uint32[2]", // Multiplies rewards for orders of the material, consists of numerator and denominator
+                name: "string",
+            }
+        },
```

Comment:
`name` being here is kind of a hack, because interacting with token contracts client-side is currently too complicated
You could add `description`, `image` or whatever else here as well, though ideally there is an erc-20 extension for that, but extending the mud module is yet another needless complexity currently